lib/pre_assembly/digital_object.rb

```diff
@@ -198,74 +178,6 @@ def container_basename
     # Registration and other Dor interactions.
     ####
 
-    def register
-      return unless project_style[:should_register]
-      log "    - register(#{pid})"
-      self.dor_object = register_in_dor(registration_params)
-      self.reg_by_pre_assembly = true
-    end
-
-    def register_in_dor(params)
-      with_retries(max_tries: Dor::Config.dor.num_attempts, rescue: Exception, handler: PreAssembly.retry_handler('REGISTER_IN_DOR', method(:log), params)) do
```

Comment:
I think this was the only place `PreAssembly.retry_handler` was used, so it could be removed too.

Reply:
Good point, thx.